sphinx/util/docutils.py

```diff
@@ -12,9 +12,13 @@
 
 from copy import copy
 from contextlib import contextmanager
+import docutils
 from docutils.parsers.rst import directives, roles
 
 
+__version__  = tuple(map(int, docutils.__version__.split('.')))
```

Comment:
I think the convention is to call the version tuple `__version_info__`, isn't it?